{"level":"error","message":"Server error: PrismaClientInitializationError: \nInvalid `prisma.user.findFirst()` invocation:\n\n\nCan't reach database server at `localhost:5432`\n\nPlease make sure your database server is running at `localhost:5432`.\n    at Mn.handleRequestError (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:121:8053)\n    at Mn.handleAndLogRequestError (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:121:7061)\n    at Mn.request (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:121:6745)\n    at async l (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:130:9633)\n    at async login (file:///home/lucifer/projects/messApp/api/controllers/authController.js:63:18) {\n  [stack]: 'PrismaClientInitializationError: \\n' +\n    'Invalid `prisma.user.findFirst()` invocation:\\n' +\n    '\\n' +\n    '\\n' +\n    \"Can't reach database server at `localhost:5432`\\n\" +\n    '\\n' +\n    'Please make sure your database server is running at `localhost:5432`.\\n' +\n    '    at Mn.handleRequestError (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:121:8053)\\n' +\n    '    at Mn.handleAndLogRequestError (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:121:7061)\\n' +\n    '    at Mn.request (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:121:6745)\\n' +\n    '    at async l (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:130:9633)\\n' +\n    '    at async login (file:///home/lucifer/projects/messApp/api/controllers/authController.js:63:18)',\n  [message]: '\\n' +\n    'Invalid `prisma.user.findFirst()` invocation:\\n' +\n    '\\n' +\n    '\\n' +\n    \"Can't reach database server at `localhost:5432`\\n\" +\n    '\\n' +\n    'Please make sure your database server is running at `localhost:5432`.',\n  name: 'PrismaClientInitializationError',\n  clientVersion: '5.21.1',\n  errorCode: undefined,\n  [Symbol(Symbol.toStringTag)]: [Getter]\n}","service":"user-service","timestamp":"2024-10-31T21:37:47.974Z"}
{"level":"error","message":"Server error: PrismaClientInitializationError: \nInvalid `prisma.user.findFirst()` invocation:\n\n\nCan't reach database server at `localhost:5432`\n\nPlease make sure your database server is running at `localhost:5432`.\n    at Mn.handleRequestError (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:121:8053)\n    at Mn.handleAndLogRequestError (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:121:7061)\n    at Mn.request (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:121:6745)\n    at async l (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:130:9633)\n    at async login (file:///home/lucifer/projects/messApp/api/controllers/authController.js:63:18) {\n  [stack]: 'PrismaClientInitializationError: \\n' +\n    'Invalid `prisma.user.findFirst()` invocation:\\n' +\n    '\\n' +\n    '\\n' +\n    \"Can't reach database server at `localhost:5432`\\n\" +\n    '\\n' +\n    'Please make sure your database server is running at `localhost:5432`.\\n' +\n    '    at Mn.handleRequestError (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:121:8053)\\n' +\n    '    at Mn.handleAndLogRequestError (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:121:7061)\\n' +\n    '    at Mn.request (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:121:6745)\\n' +\n    '    at async l (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:130:9633)\\n' +\n    '    at async login (file:///home/lucifer/projects/messApp/api/controllers/authController.js:63:18)',\n  [message]: '\\n' +\n    'Invalid `prisma.user.findFirst()` invocation:\\n' +\n    '\\n' +\n    '\\n' +\n    \"Can't reach database server at `localhost:5432`\\n\" +\n    '\\n' +\n    'Please make sure your database server is running at `localhost:5432`.',\n  name: 'PrismaClientInitializationError',\n  clientVersion: '5.21.1',\n  errorCode: undefined,\n  [Symbol(Symbol.toStringTag)]: [Getter]\n}","service":"user-service","timestamp":"2024-11-01T17:18:19.337Z"}
{"level":"error","message":"Validation errors: [\n  {\n    type: 'field',\n    value: { identifier: 'user2', password: '123456' },\n    msg: 'Email or username is required',\n    path: '',\n    location: 'body'\n  },\n  [length]: 1\n]","service":"user-service","timestamp":"2024-11-04T10:16:16.989Z"}
{"level":"error","message":"Validation errors: [\n  {\n    type: 'field',\n    value: '123',\n    msg: 'Password must be at least 6 characters',\n    path: 'password',\n    location: 'body'\n  },\n  [length]: 1\n]","service":"user-service","timestamp":"2024-11-06T22:05:16.472Z"}
{"level":"error","message":"Server error: PrismaClientInitializationError: \nInvalid `prisma.user.findFirst()` invocation:\n\n\nCan't reach database server at `localhost:5432`\n\nPlease make sure your database server is running at `localhost:5432`.\n    at Mn.handleRequestError (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:121:8053)\n    at Mn.handleAndLogRequestError (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:121:7061)\n    at Mn.request (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:121:6745)\n    at async l (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:130:9633)\n    at async login (file:///home/lucifer/projects/messApp/api/controllers/authController.js:63:18) {\n  [stack]: 'PrismaClientInitializationError: \\n' +\n    'Invalid `prisma.user.findFirst()` invocation:\\n' +\n    '\\n' +\n    '\\n' +\n    \"Can't reach database server at `localhost:5432`\\n\" +\n    '\\n' +\n    'Please make sure your database server is running at `localhost:5432`.\\n' +\n    '    at Mn.handleRequestError (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:121:8053)\\n' +\n    '    at Mn.handleAndLogRequestError (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:121:7061)\\n' +\n    '    at Mn.request (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:121:6745)\\n' +\n    '    at async l (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:130:9633)\\n' +\n    '    at async login (file:///home/lucifer/projects/messApp/api/controllers/authController.js:63:18)',\n  [message]: '\\n' +\n    'Invalid `prisma.user.findFirst()` invocation:\\n' +\n    '\\n' +\n    '\\n' +\n    \"Can't reach database server at `localhost:5432`\\n\" +\n    '\\n' +\n    'Please make sure your database server is running at `localhost:5432`.',\n  name: 'PrismaClientInitializationError',\n  clientVersion: '5.21.1',\n  errorCode: undefined,\n  [Symbol(Symbol.toStringTag)]: [Getter]\n}","service":"user-service","timestamp":"2024-11-07T16:25:10.669Z"}
{"level":"error","message":"Server error: PrismaClientInitializationError: \nInvalid `prisma.user.findFirst()` invocation:\n\n\nCan't reach database server at `localhost:5432`\n\nPlease make sure your database server is running at `localhost:5432`.\n    at Mn.handleRequestError (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:121:8053)\n    at Mn.handleAndLogRequestError (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:121:7061)\n    at Mn.request (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:121:6745)\n    at async l (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:130:9633)\n    at async login (file:///home/lucifer/projects/messApp/api/controllers/authController.js:63:18) {\n  [stack]: 'PrismaClientInitializationError: \\n' +\n    'Invalid `prisma.user.findFirst()` invocation:\\n' +\n    '\\n' +\n    '\\n' +\n    \"Can't reach database server at `localhost:5432`\\n\" +\n    '\\n' +\n    'Please make sure your database server is running at `localhost:5432`.\\n' +\n    '    at Mn.handleRequestError (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:121:8053)\\n' +\n    '    at Mn.handleAndLogRequestError (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:121:7061)\\n' +\n    '    at Mn.request (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:121:6745)\\n' +\n    '    at async l (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:130:9633)\\n' +\n    '    at async login (file:///home/lucifer/projects/messApp/api/controllers/authController.js:63:18)',\n  [message]: '\\n' +\n    'Invalid `prisma.user.findFirst()` invocation:\\n' +\n    '\\n' +\n    '\\n' +\n    \"Can't reach database server at `localhost:5432`\\n\" +\n    '\\n' +\n    'Please make sure your database server is running at `localhost:5432`.',\n  name: 'PrismaClientInitializationError',\n  clientVersion: '5.21.1',\n  errorCode: undefined,\n  [Symbol(Symbol.toStringTag)]: [Getter]\n}","service":"user-service","timestamp":"2024-11-07T16:25:16.905Z"}
{"level":"error","message":"Server error: PrismaClientInitializationError: \nInvalid `prisma.user.findFirst()` invocation:\n\n\nCan't reach database server at `localhost:5432`\n\nPlease make sure your database server is running at `localhost:5432`.\n    at Mn.handleRequestError (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:121:8053)\n    at Mn.handleAndLogRequestError (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:121:7061)\n    at Mn.request (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:121:6745)\n    at async l (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:130:9633)\n    at async login (file:///home/lucifer/projects/messApp/api/controllers/authController.js:63:18) {\n  [stack]: 'PrismaClientInitializationError: \\n' +\n    'Invalid `prisma.user.findFirst()` invocation:\\n' +\n    '\\n' +\n    '\\n' +\n    \"Can't reach database server at `localhost:5432`\\n\" +\n    '\\n' +\n    'Please make sure your database server is running at `localhost:5432`.\\n' +\n    '    at Mn.handleRequestError (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:121:8053)\\n' +\n    '    at Mn.handleAndLogRequestError (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:121:7061)\\n' +\n    '    at Mn.request (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:121:6745)\\n' +\n    '    at async l (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:130:9633)\\n' +\n    '    at async login (file:///home/lucifer/projects/messApp/api/controllers/authController.js:63:18)',\n  [message]: '\\n' +\n    'Invalid `prisma.user.findFirst()` invocation:\\n' +\n    '\\n' +\n    '\\n' +\n    \"Can't reach database server at `localhost:5432`\\n\" +\n    '\\n' +\n    'Please make sure your database server is running at `localhost:5432`.',\n  name: 'PrismaClientInitializationError',\n  clientVersion: '5.21.1',\n  errorCode: undefined,\n  [Symbol(Symbol.toStringTag)]: [Getter]\n}","service":"user-service","timestamp":"2024-11-07T16:25:23.122Z"}
{"level":"error","message":"Server error: RangeError: Maximum call stack size exceeded\n    at mp (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:23:909)\n    at js (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:23:826)\n    at qs (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:23:119)\n    at js (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:23:851)\n    at qs (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:23:119)\n    at js (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:23:851)\n    at qs (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:23:119)\n    at js (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:23:851)\n    at qs (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:23:119)\n    at js (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:23:851) {\n  [stack]: 'RangeError: Maximum call stack size exceeded\\n' +\n    '    at mp (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:23:909)\\n' +\n    '    at js (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:23:826)\\n' +\n    '    at qs (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:23:119)\\n' +\n    '    at js (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:23:851)\\n' +\n    '    at qs (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:23:119)\\n' +\n    '    at js (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:23:851)\\n' +\n    '    at qs (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:23:119)\\n' +\n    '    at js (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:23:851)\\n' +\n    '    at qs (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:23:119)\\n' +\n    '    at js (/home/lucifer/projects/messApp/api/node_modules/@prisma/client/runtime/library.js:23:851)',\n  [message]: 'Maximum call stack size exceeded',\n  clientVersion: '5.21.1'\n}","service":"user-service","timestamp":"2024-11-15T15:13:32.076Z"}
{"level":"error","message":"Server error: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (node:_http_outgoing:659:11)\n    at ServerResponse.header (/home/lucifer/projects/messApp/api/node_modules/express/lib/response.js:794:10)\n    at ServerResponse.send (/home/lucifer/projects/messApp/api/node_modules/express/lib/response.js:174:12)\n    at ServerResponse.json (/home/lucifer/projects/messApp/api/node_modules/express/lib/response.js:278:15)\n    at getOnlineUsers (file:///home/lucifer/projects/messApp/api/controllers/authController.js:172:9) {\n  [stack]: 'Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\\n' +\n    '    at ServerResponse.setHeader (node:_http_outgoing:659:11)\\n' +\n    '    at ServerResponse.header (/home/lucifer/projects/messApp/api/node_modules/express/lib/response.js:794:10)\\n' +\n    '    at ServerResponse.send (/home/lucifer/projects/messApp/api/node_modules/express/lib/response.js:174:12)\\n' +\n    '    at ServerResponse.json (/home/lucifer/projects/messApp/api/node_modules/express/lib/response.js:278:15)\\n' +\n    '    at getOnlineUsers (file:///home/lucifer/projects/messApp/api/controllers/authController.js:172:9)',\n  [message]: 'Cannot set headers after they are sent to the client',\n  code: 'ERR_HTTP_HEADERS_SENT',\n  [Symbol(kIsNodeError)]: [Getter]\n}","service":"user-service","timestamp":"2024-11-15T15:41:02.691Z"}
